components/02-molecules/menus/_menu-item.twig

```diff
@@ -25,7 +25,7 @@
 {% import "@molecules/menus/_menu.twig" as menus %}
```

Comment:
Can't comment on lines you didn't change but starting on line 21 we should also change these to be `list__item__base_class` `list__item__modifiers` `list__item__blockname` as well.